src/Page/Textile/Simulator.elm

```diff
@@ -614,24 +629,44 @@ massField massInput =
         ]
 
 
-productField : Inputs.Query -> Html Msg
-productField query =
+durabilityField : (Unit.Durability -> Msg) -> Unit.Durability -> Html Msg
+durabilityField updateDurability durability =
     let
-        autocompleteState =
-            AutocompleteSelector.init Inputs.exampleProductToString Inputs.exampleProducts
+        fromFloat =
+            Unit.durabilityToFloat >> String.fromFloat
     in
-    div []
-        [ label
-            [ for "selector-example"
-            , class "form-label fw-bold"
-            ]
-            [ text "Produit" ]
-        , button
-            [ class "form-select ElementSelector text-start"
-            , id "selector-example"
-            , onClick (SetModal (SelectExampleModal autocompleteState))
+    div [ class "d-block" ]
+        [ label [ for "durability-field", class "form-label fw-bold text-truncate" ]
+            [ text "Durabilité" ]
+        , div [ class "d-flex justify-content-between gap-3 mt-2" ]
+            [ input
+                [ type_ "range"
+                , id "durability-field"
+                , class "d-block form-range"
+                , title "Un double-clic réinitialise la valeur"
+                , onInput
+                    (String.toFloat
+                        >> Maybe.map Unit.durability
+                        >> Maybe.withDefault Unit.standardDurability
+                        >> updateDurability
+                    )
+                , onDoubleClick (updateDurability Unit.standardDurability)
```

Comment:
Clever, even if maybe redundant with the "reset" button (admitedly very far at the bottom).
Should we replicate this UX on all the sliders, in a future PR? If so, let's create a Notion card?

Reply:
This is the behavior in a few software I use, eg. Adobe Lightroom, where double-clicking a range slider resets it. I'm not entirely sure it's a general well-known behavior so I'm not sure if we should generalize it or not… I've created [a card](https://www.notion.so/G-n-raliser-la-r-initialisation-au-double-click-sur-les-rangesliders-227ae641eb8e4cfa9fdbce502410714b) about it.